e2e-tests/tests/test_e2e_integration_flows.py

```diff
@@ -239,3 +241,29 @@ def test_demo_website_text_returns_correct_anonymized_version():
     # Assert equal
 
     assert expected_anonymized_text == actual_anonymized_text
+
+
+@pytest.mark.package
+def test_given_text_with_pii_using_package_then_analyze_and_anonymize_complete_successfully():
+    analyzer_request = {
+        "text": "John Smith drivers license is AC432223",
+        "language": "en",
+    }
+
+    expected_response = [
```

Comment:
How about taking some of the examples we have in the samples and turning them into tests? Less json oriented and more python development oriented. In addition we should have maybe some additional tests with different parameters, making sure things to break in other scenarios, for example when the analyzer engine output is empty, or when we have intersecting entities detected.

Reply:
The `en_core_web_sm` didn't work because it looks like some other tests are looking for the LG one, and they failed. 

Don't we have unit tests that cover such regression cases (empty output, intersections etc)? I think that following the spirit of e2e tests we are not looking for fail scenarios.

For an e2e scenario I would choose one "good day" path. If the samples provide other "good day" scenarios that are meaningful, we can add it.

Reply:
I converted the json into objects